site/en/reference/cli/contribute.md

```diff
@@ -0,0 +1,298 @@
+# Update OT CLI Commands
```

Comment:
One thing I just thought of, maybe have a note that if the user needs help/guidance on updating the CLI commands, to ask on the community.  I think it might be tricky for some given how the documentation is in the cli code itself.